frontend/src/pages/admin/Stacks/dialogs/AdminStackEditDialog.vue

```diff
@@ -18,7 +19,7 @@
         </template>
         <template v-slot:actions>
             <ff-button kind="secondary" @click="close()">Cancel</ff-button>
-            <ff-button kind="primary" @click="confirm()" :disabled="!formValid">{{ (stack ? 'Save' : 'Create') }}</ff-button>
+            <ff-button kind="primary" @click="confirm()" :disabled="!formValid || !loading">{{ (stack ? 'Save' : 'Create') }}</ff-button>
```

Comment:
Missing this whilst reviewing - the logic is flipped here. Should be `loading` not `!loading`.

Reply:
sorry, just realised th diff you're showing there is my change, not your requested change. Just approved the other PR (https://github.com/flowforge/flowforge/pull/659#pullrequestreview-997641914)